conan/internal/api/new/basic.py

```diff
@@ -1,13 +1,24 @@
+def inject_get_or_else(variable, default):
```

Comment:
There is a way to create functions in jinja templates directly, but I am fine with this too.